Linaro/qa-reports-known-issues
# Features and refactors

- Add 'test_names' list field
- Add 'matrix_apply' option
- Add a check to ensure each test is defined once
- Add several tests
- If a project doesn't exist in qa-reports, fail
